src/index.ts

```diff
@@ -40,9 +41,25 @@ async function asyncServerless() {
     });
 }
 
+async function checkPatientOrgsClaim(events: any) {
+    const decodedToken: any = jwt_decode(events.headers.Authorization.replace('Bearer ', ''));
+    if (
+        events.path === '/DetectedIssue' &&
+        events.httpMethod === 'GET' &&
+        decodedToken.patientOrgs &&
+        decodedToken.fhirUser &&
+        decodedToken.scp.some((scope: any) => scope.startsWith('user/'))
+    ) {
+        events.queryStringParameters = { 'patient:Patient.organization': decodedToken.patientOrgs };
+        events.multiValueQueryStringParameters = { 'patient:Patient.organization': [decodedToken.patientOrgs] };
```

Comment:
If the original query is `GET [baseUrl]/DetectedIssue?identified=ge2010-01-01`
Would `events.queryStringParameters` be `{'identified': 'ge2010-01-01'}` ?
And the above code just overrides the query string. Does it mean the actual query becomes `GET [baseUrl]/DetectedIssue?patient.organization=Organization/org-1' ? 
Shouldn't it be  `GET [baseUrl]/DetectedIssue?identified=ge2010-01-01&patient.organization=Organization/org-1' ?

I'm trying to test it on AWS Sandbox account. But It takes me much time to figure out how to get token via postman.

Reply:
That's a good question. Right now, [checkPatientOrgsClaim](https://github.com/nirojshrestha019/fhir-works-on-aws-deployment/blob/smart-mainline/src/index.ts#L44) function only checks if we have requested GET DetectedIssue resources and if patietOrgs claim is present in token. Then this function will add the chain search parameter according to only the patientOrgs claim from the token. 
To accomodate the additional search parameter like ?identified=ge2010-01-01, we can make small adjustment in that function itself by instead of replacing the whole queryStringParameter we can append the patientOrgs claim to become: [baseUrl]/DetectedIssue?identified=ge2010-01-01&patient.organization=patientOrgs_claim.